### PR TITLE
add requirements

### PR DESCRIPTION
**Pull request by voroyam**
In order to setup a OfficeOnline Server, there were requirements missing. I added them

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 1]
```diff
@@ -22,6 +22,11 @@ Please bear in mind:
 
 TIP: If you want to integrate the {office365-url}[Office 365 (cloud)] version of Microsoft Office Online, you need to {oc-contact-url}[get in touch with us].
 
+== Preparing the Environment
+You need an https://docs.microsoft.com/de-de/officeonlineserver/deploy-office-online-server[OfficeOnline Server] installed.
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: @voroyam info: please create the link in the form as it is made in the TIP above = definition on top and reference usage in the line. We have this procedure in all documents for better readability and managability.

Comment (voroyam) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: done

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 2]
```diff
@@ -3,6 +3,7 @@
 :msoffice-online-server-url: https://www.microsoft.com/en-us/microsoft-365/blog/2016/05/04/office-online-server-now-available/
 :office365-url: https://products.office.com/en-us/business/office 
 :wopi-protocol-url: https://wopi.readthedocs.io/en/latest/
+:officeonline-server: https://docs.microsoft.com/de-de/officeonlineserver/deploy-office-online-server
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: I know it is a pain, but you need to name it with ending -url.
This helps to distinguish on the fly if the name references a url or not.

Comment (voroyam) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: No worries. I did not see the other links having the -url ending.

added

**Comment by mmattel** [comment]
Please put a note to which release(s) a backport is needed

**Comment by voroyam** [comment]
Needs to be backported to all the current branches. 10.4. 10.3

**Pull request merged**